### PR TITLE
webhooks: Small cleanups

### PR DESCRIPTION
**Pull request by ryanslade**
Removed some dead code and simplified the signature of a method. Just a
few small things I spotted while thinking about how we can simply
webhook setup.

## Test plan

Existing tests

### CODE REVIEW

File: cmd/frontend/internal/httpapi/httpapi.go [hunk 1]
```diff
@@ -90,13 +90,12 @@ func NewHandler(
...
 		DB: db,
 	}
-	webhookhandlers.Init(db, &wh)
+	webhookhandlers.Init(&wh)
 	handlers.BatchesGitHubWebhook.Register(&wh)
 	handlers.BatchesGitLabWebhook.Register(&wh)
 	handlers.GitHubSyncWebhook.Register(&wh)
 
 	// 🚨 SECURITY: This handler implements its own secret-based auth
-	// TODO: Integrate with webhookMiddleware.Logger
```
Comment (sashaostrikov) [inline cmd/frontend/internal/httpapi/httpapi.go:99]: I guess we are not making this integration? 😁

**Review by sashaostrikov — APPROVED** [review]
LGTM!

File: cmd/frontend/internal/httpapi/httpapi.go (hunk 1 above)
Comment (ryanslade) [reply cmd/frontend/internal/httpapi/httpapi.go:99]: It's already done but we forgot to remove the TODO.

See a few lines lower down where we wrap it in the logging middleware

**Review by mollylogue — COMMENTED** [review]
🙏

**Pull request merged**